code/game/machinery/announcement_system.dm

```diff
@@ -172,7 +172,7 @@ GLOBAL_LIST_EMPTY(announcement_systems)
 		return
 
 	arrival = pick("#!@%ERR-34%2 CANNOT LOCAT@# JO# F*LE!", "CRITICAL ERROR 99.", "ERR)#: DA#AB@#E NOT F(*ND!")
-	newhead = pick("OV#RL()D: \[UNKNOWN??\] DET*#CT)D!", "ER)#R - B*@ TEXT F*O(ND!", "AAS.exe is not responding. NanoOS is searching for a solution to the problem.")
+	newhead = pick("OV#RL()D: \[UNKNOWN??] DET*#CT)D!", "ER)#R - B*@ TEXT F*O(ND!", "AAS.exe is not responding. NanoOS is searching for a solution to the problem.")
```

Comment:
this looks like a mistake

Reply:
It's not, \] is not required

Reply:
does it hurt anything? the consistency seems nice. also like why this pr